esp_idf_panic_decoder/pc_address_decoder.py

```diff
@@ -23,50 +30,136 @@ def __init__(
         self.toolchain_prefix = toolchain_prefix
         self.elf_files = elf_file if isinstance(elf_file, list) else [elf_file]
         self.rom_elf_file = rom_elf_file
-        self.pc_address_buffer = b''
         self.pc_address_matcher = [PcAddressMatcher(file) for file in self.elf_files]
-        if rom_elf_file is not None:
-            self.rom_pc_address_matcher = PcAddressMatcher(rom_elf_file)
+        if self.rom_elf_file:
+            self.pc_address_matcher.append(PcAddressMatcher(self.rom_elf_file))
+
+    def decode_address(self, line: bytes):
```

Comment:
The naming of this function is a little bit unfortunate because we have also a `decode_addresses()` and the comment is not reflecting the difference between them.